deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +254,143 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payload: __ENV.DATA_BTM_SQS_PAYLOAD,
+  payloadData1: __ENV.DATA_BTM_SQS_PAYLOAD_DATA1,
+  payloadData2: __ENV.DATA_BTM_SQS_PAYLOAD_DATA2,
+  payloadData3: __ENV.DATA_BTM_SQS_PAYLOAD_DATA3,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadData1Array = eventData.payloadData1.split(',')
+const payloadData2Array = eventData.payloadData2.split(',')
+const payloadData3Array = eventData.payloadData3.split(',')
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
+let messageBody: string
+let randomTimestamp: number
+let timestampFormatted: string
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
+export function sendEvent1 (): void {
```

Comment:
`sendEvent1` to `sendEvent7` seem to have massive code duplication. Can you not parameterise?